src/components/ReactTags.js

```diff
@@ -89,12 +93,16 @@ class ReactTags extends Component {
       suggestions,
       query: '',
       isFocused: false,
+      editing: false,
+      editValue: null,
+      editIndex: null,
```

Comment:
this is pointing to the current editable index right ? so can we rename this to currentEditIndex ?